html-forms/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+
+  <form action="" method="get">
+    <label for="name">Email<br /></label>
+    <input id="name" type="text" name="email">
+    <br />
+
+
+    <label for="drop">Why is coding cool?<br /></label>
+    <select id="drop" name="reasons">
+      <option value="option">Select an option</option>
+      <option value="interesting">Because it's interesting</option>
+      <option value="fun">Because it's fun</option>
+      <option value="powerful">Because it's powerfun</option>
+    </select>
+    <br />
+    <br />
+
+
+    <div>Favorite Language (choose one)<br /><br /></div>
+    <input id="HTML" type="radio" name="language" value="HTML">
+    <label for="HTML">HTML</label>
+    <input id="CSS" type="radio" name="language" value="CSS">
+    <label for="CSS">CSS</label>
+    <input id="JavaScript" type="radio" name="language" value="Javascript">
+    <label for="Javascript">Javascript</label>
+    <br />
+    <br />
+
+
+    <div>Would you like to sign up for more information?<br /><br /></div>
+    <input id="signup" type="checkbox" name="signup">
+    <label for="signup">Sign me up!</label><br />
+
+
+    <div>Comment</div>
+    <textarea id="comments" name="comments" cols="20" rows="5"></textarea>
+    <label for="comments"></label>
```

Comment:
the text `comment` should be inside of the label so that it can be used to access the input